patches/yjs@13.5.43.patch

```diff
@@ -0,0 +1,217 @@
+diff --git a/dist/src/types/YMap.d.ts b/dist/src/types/YMap.d.ts
+index 00b8e8178827727172ed32930e6c76c1b1842f64..b8daa1898734caf05c2fe5807a98753978615e4d 100644
+--- a/dist/src/types/YMap.d.ts
++++ b/dist/src/types/YMap.d.ts
```

Comment:
If other future y-provider packages (which may be maintained by us) depend on a non-patched version, will this lead to two different versions of yjs?

Or else maybe we can make these packages depend on `@blocksuite/store`, and get yjs from the static getter at `Workspace.Y`?

Reply:
if downstream use Workspace.Y, they dont have this patch